packages/reaction-core/common/methods/cart.js

```diff
@@ -4,8 +4,9 @@
 //
 if (Meteor.isClient) {
   Meteor.methods({
-    "cart/submitPayment": function (paymentMethod) {
+    "cart/submitPayment": function (paymentMethod, sessionId) {
```

Comment:
we can have the same method defined  on the server and client.  We can have the sessionId as param on server, but if we avoid requiring it on the client and use `Sessions` then the payment packages (which both are calling submit as a client methods) wouldn't need to pass sessionId.  I think it's a worthwhile change, even if a bit of a confusing implementation if we can avoid the breaking changes to the paymentSubmit method..